htdocs/compta/facture/class/facture.class.php

```diff
@@ -1275,6 +1275,13 @@ public function createFromClone(User $user, $fromid = 0)
 			$object->lines[$i]->ref_ext = ''; // Do not clone ref_ext
 		}
 
+		if ($objFrom->socid != $object->socid) {
```

Comment:
Original idea of clone was "We make a similar copy".
This means we probably need a feature instead. When we clone, we can have into the confirmation request a checkox
"Modify the VAT rate to use the default rate depending on the new thirdparty".

I suggest to provide a PR with such an enhancement so we can keep the rule "Clone is a mirror except for option chosen" consistent everywhere